lib/formats/bs_format_ripejson.c

```diff
@@ -220,6 +226,46 @@ static int process_common_fields(bgpstream_format_t *format,
   return 0;
 }
 
+// readline
+static int readline(bgpstream_format_t *format)
+{
+  int total_read = 0;
+  int current_read = 0;
+  char *buffer_ptr = STATE->json_string_buffer;
+
+  while(1){
+    // read line, including linebreak
+    current_read = bgpstream_transport_readline(
+      format->transport, buffer_ptr, BUFSIZE - total_read, 0);
+    total_read+=current_read;
+
+    if(current_read == 0){
```

Comment:
what if this is not the first iteration of the loop?
don't you want to return total_read instead of 0?

Reply:
You are right. Fixed in da4845c